
Juck - let's build the build helpers using the system compiler.

  - Rene Rebe <rene@exactcode.de>

diff -ur ash-linux-0.2/Makefile ash-linux-0.2-fixed/Makefile
--- ash-linux-0.2/Makefile	1993-10-04 19:48:13.000000000 +0100
+++ ash-linux-0.2-fixed/Makefile	2005-02-02 11:02:26.198007168 +0100
@@ -40,19 +42,19 @@
 	touch init.c
 
 mkinit: mkinit.c
-	$(CC) $(CFLAGS) $(LDFLAGS) mkinit.c -o $@ $(LDADD)
+	$(BUILDCC) $(CFLAGS) $(LDFLAGS) mkinit.c -o $@ $(LDADD)
 
 nodes.c nodes.h: mknodes nodetypes nodes.c.pat
 	./mknodes nodetypes nodes.c.pat
 
 mknodes: mknodes.c
-	$(CC) $(CFLAGS) $(LDFLAGS) mknodes.c -o $@ $(LDADD)
+	$(BUILDCC) $(CFLAGS) $(LDFLAGS) mknodes.c -o $@ $(LDADD)
 
 syntax.c syntax.h: mksyntax
 	./mksyntax
 
 mksyntax: mksyntax.c parser.h
-	$(CC) $(CFLAGS) $(LDFLAGS) mksyntax.c -o $@ $(LDADD)
+	$(BUILDCC) $(CFLAGS) $(LDFLAGS) mksyntax.c -o $@ $(LDADD)
 
 clean:
 	rm -f core $(CLEANFILES) $(PROG) $(OBJS)
